README.md

```diff
@@ -1,33 +1,46 @@
 # Blender URDF Viewer
 
-v0.1.2
+v0.1.3
 
 Tested with Blender v2.90.1, v2.93.4
 
+Available Add-ons:
+- URDF visualizer
+- Batch mesh decimator
+
 Often when I'm generating URDFs from SolidWorks I need a visualizer where I can move joints and links around and also do things like decimate the meshes so they are a reasonable file size. This is a collection of those Blender scripts that I've found helpful while creating and working with URDF packages.
 
+## URDF Visualizer
 ![GIF of Blender URDF visualization](./images/yumi.gif)
 
-## Current Limitations
+### Current Limitations
 
 - Only Fixed, Revolute, Continuous, and Prismatic joint types.
 - Doesn't handle primitives. Assumes you're working with generated mesh files.
 - Cannot save configuration.
 - Cannot load URDFs that have models in different packages.
 
-**NOTES:**
-1. During development, you may need to disable the Add-on and re-enable it for it to load properly.
-2. To see all of the output from the console, launch blender from the command line.
+## Mesh Decimator
+
+This does not produce any visual results, but allows you to select a directory of meshes and specify the maxium size that you'd like each mesh to be. The script will then automatically look for STL files, reduce their size if necessary, and re-export to get rid of the `SOLID` warning Rviz throws for STLs generated with SolidWorks.
+
+Backups of all files are created (using a `*.1`, `*.2`, etc) modifier to the filename.
```

Comment:
Maybe I just need to learn how to use blender, but it's not immediately obvious to me how to run the decimator add-on after enabling it in preferences. Consider adding a sentence on that?

Reply:
Added a gif for how to enable add-ons to the README